src/Illuminate/Support/Str.php

```diff
@@ -472,6 +472,19 @@ public static function unwrap($value, $before, $after = null)
         return $value;
     }
 
+    /**
+     * Formats a given string into a valid PAN (Primary Account Number, which is used for  e.g. IBAN and credit card numbers).
+     *
+     * @param  string  $value
+     * @return string
+     */
+    public static function pan($value)
+    {
+        $value = (string) $value;
+
+        return join(' ', str_split($value, 4));
```

Comment:
```suggestion
        return implode(' ', str_split($value, 4));
```

Laravel may have `Collection::join()` and `Arr::join()`. But when using PHP built-in function we always opt for `implode()` as `join()` is an alias: https://www.php.net/manual/en/function.join.php

Reply:
Thank you. 😄